homeassistant/components/tibber/sensor.py

```diff
@@ -59,6 +60,7 @@ def __init__(self, tibber_home):
             self._name = tibber_home.info["viewer"]["home"]["address"].get(
                 "address1", ""
             )
+        self._spread_load_constant = randrange(3600)
```

Comment:
Codacy found an issue: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)